test-framework/core/src/main/java/org/keycloak/testframework/server/AbstractKeycloakServerSupplier.java

```diff
@@ -20,23 +23,25 @@
 public abstract class AbstractKeycloakServerSupplier implements Supplier<KeycloakServer, KeycloakIntegrationTest> {
 
     @Override
-    public KeycloakServer getValue(InstanceContext<KeycloakServer, KeycloakIntegrationTest> instanceContext) {
-        KeycloakIntegrationTest annotation = instanceContext.getAnnotation();
-        KeycloakServerConfig serverConfig = SupplierHelpers.getInstance(annotation.config());
-
-        KeycloakServerConfigBuilder command = KeycloakServerConfigBuilder.startDev()
-                .bootstrapAdminClient(Config.getAdminClientId(), Config.getAdminClientSecret())
-                .bootstrapAdminUser(Config.getAdminUsername(), Config.getAdminPassword());
+    public List<Dependency> getDependencies(RequestedInstance<KeycloakServer, KeycloakIntegrationTest> instanceContext) {
+        KeycloakServerConfigBuilder command = getKeycloakServerConfigBuilder(instanceContext.getAnnotation());
 
-        command.log().handlers(KeycloakServerConfigBuilder.LogHandlers.CONSOLE);
+        DependenciesBuilder builder = DependenciesBuilder.create(ManagedCertificates.class).add(ManagedCertificates.class);
```

Comment:
double times ManagedCertificates -> one of them should be most probably AdminClient?

Reply:
Removed duplicate. It should not be `AdminClient` as the server does not have a dependency on that.